client/index.html

```diff
@@ -241,29 +241,28 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<h2>Notifications</h2>
 							</div>
-							<div class="col-sm-12">
+							<div class="col-sm-6">
 								<label class="opt">
 								<input id="badge" type="checkbox" name="badge">
 								Enable badge
 								</label>
-							</div>
-							<div class="col-sm-12">
```

Comment:
Yeah, I think both of them are ugly, but I prefer it after my change. So if no one else has a big objection, I'll keep that in this PR.

Yeah, we really should look at the UI/UX of the settings in general. Could be done in conjunction with adding server syncing (we could have optional overrides for individual clients for example).

But let's leave that to the other discussion. I'll add an issue if no one else has.